packages/origin-backend/readme.md

```diff
@@ -0,0 +1,5 @@
+# Energy Web Foundation: origin-release C: origin-backend 
```

Comment:
I'm not sure if adding mention about `release C` to origin-backend readme adds valuable information to the document, maybe we can omit this information?